backend/pipeline/serializers/crud.py

```diff
@@ -1,13 +1,15 @@
 import logging
 from collections import OrderedDict
-from typing import Any
+from typing import Any, Optional
 
 from connector.connector_instance_helper import ConnectorInstanceHelper
 from connector.models import ConnectorInstance
 from connector_processor.connector_processor import ConnectorProcessor
+from croniter import croniter
```

Comment:
@muhammad-ali-e is this a new library? If yes, please add it to `pyproject.toml`

Reply:
Oh... It was there, For got to commit

Reply:
Added